src/Catalyst.Core.Lib/Kernel/Kernel.cs

```diff
@@ -177,9 +178,9 @@ public Kernel WithPersistenceConfiguration(string configSection = null)
         ///     Allows custom nodes to write custom code for containerBuilder
         /// </summary>
         /// <param name="customBootLogic"></param>
-        public void StartCustom(CustomBootLogic customBootLogic)
+        public async Task StartCustomAsync(CustomBootLogic customBootLogic)
         {
-            customBootLogic.Invoke(this);
+            await customBootLogic.Invoke(this);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4343762)